pkg/process/util/util.go

```diff
@@ -123,6 +123,14 @@ func GetSysRoot() string {
 	return "/sys"
 }
 
+// GetRootNSPID returns the current PID from the root namespace
+func GetRootNSPID() int {
+	pidPath := filepath.Join(GetProcRoot(), "self")
+	pidStr, _ := os.Readlink(pidPath)
```

Comment:
I think I'd prefer to return `0, err` when these functions error. The caller can always ignore the error, if they want.

Reply:
ok. I thought of this as a drop-in replacement for `os.Getpid`, but I think it's fair to return an error.

Reply:
done